bm-persona/Map/MapDataSource/MapMarker.swift

```diff
@@ -35,37 +39,63 @@ enum MapMarkerType: String, CaseIterable {
         case .microwave:
             icon = UIImage(named: "microwave-icon")
             break
-        case .napPod:
-            icon = UIImage(named: "nap-pods-icon")
+        case .rest:
+            icon = UIImage(named: "nap-pods-icon")?
+                .withRoundedBorder(width: 30, color: .white)?
+                .withShadow(blur: 10, offset: CGSize(width: 0, height: 10))
             break
         case .printer:
-            icon = UIImage(named: "printer-icon")
+            icon = UIImage(named: "printer-icon")?
+                .withRoundedBorder(width: 3, color: .white)?
+                .withShadow()
             break
         case .water:
             icon = UIImage(named: "water-bottle-icon")
             break
         case .bikes:
-            icon = UIImage(named: "bike-icon")
+            icon = UIImage(named: "bike-icon")?
+                .withRoundedBorder(width: 3, color: .white)?
+                .withShadow()
+            break
+        case .lactation:
+            icon = UIImage(named: "lactation-icon")
+            break
+        case .waste:
+            icon = UIImage(named: "waste-icon")
+            break
+        case .garden:
+            icon = UIImage(named: "garden-icon")
+            break
+        case .cafe:
+            icon = UIImage(named: "cafe-icon")
```

Comment:
some of these icons look kinda pixelated on my phone, is there a reason for that?

Reply:
![IMG_1722](https://user-images.githubusercontent.com/20230668/93034985-a5bffb80-f5f0-11ea-874e-1d9ff3f3d299.PNG)

Reply:
Fixed! Looks like it was a bug in `UIImage.resized(...)`